apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -175,6 +178,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     if (
         grading_method is not GradingMethodType.DAG
```

Comment:
Can replace these three `and`'s with `grading_method not in RANKING_DAG_ORDERED_GRADING_TYPE`

Reply:
Fixed in [79e31e9](https://github.com/PrairieLearn/PrairieLearn/pull/8151/commits/79e31e9ed7797f176aced8b7c3ef966f96a8a751).